appveyor.yml

```diff
@@ -1,6 +1,12 @@
 -
   os: Visual Studio 2015
 
+  branches:
+    only:
+      - /v\d\.\d\.\d/
+    except:
```

Comment:
I know we are trying to mimic existing behavior but it would be nice to run it on latest master as well so we are not forced to do a release every time

Reply:
It can be something which can be done as a follow up

Reply:
I agree, I'll try and filter only on the deploy step